tests/transactional/install_updates.pm

```diff
@@ -34,12 +34,21 @@ sub run {
     }
 
     # First we update the system
+    # By default we always use zypper patch
+    # Optionally we can also do a zypper up before
+    if (get_var('TRANSACTIONAL_UPDATE_UP')) {
```

Comment:
In what case should users perform an update and when it can be skipped?

Reply:
Some automatic services to update system have `up` by default, so we need to cover both.